packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -804,23 +806,50 @@ class RawGestureDetector extends StatefulWidget {
   /// duplication of information.
   final bool excludeFromSemantics;
 
+  /// Describes the semantics notations that should be added to the underlying
+  /// render object [RenderSemanticsGestureHandler].
+  ///
+  /// It has no effect if [excludeFromSemantics] is true.
+  ///
+  /// When [semantics] is null, [RawGestureDetector] will fall back to a
+  /// default delegate which checks if the detector owns certain gesture
+  /// recognizers and calls their callbacks if they exist.
```

Comment:
maybe change the period to a colon to make it clearer that the subsequent bulleted list is the description of the default behaviour.

Reply:
Fixed